src/wallet/rpcwallet.cpp

```diff
@@ -1335,6 +1346,24 @@ static void ListTransactions(const CWallet* const pwallet, const CWalletTx& wtx,
             ret.push_back(entry);
         }
     }
+
+    return true;
+}
+
+/**
+ * List transactions based on the given criteria.
+ *
+ * @param  pwallet        The wallet.
+ * @param  wtx            The wallet transaction.
+ * @param  nMinDepth      The minimum confirmation depth.
+ * @param  fLong          Whether to include the JSON version of the transaction.
+ * @param  ret            The UniValue into which the result is stored.
+ * @param  filter_ismine  The "is mine" filter flags.
+ * @param  filter_label   Optional label string to filter incoming transactions.
+ */
+static void ListTransactions(const CWallet* const pwallet, const CWalletTx& wtx, int nMinDepth, bool fLong, UniValue& ret, const isminefilter& filter_ismine, const std::string* filter_label)  EXCLUSIVE_LOCKS_REQUIRED(pwallet->cs_wallet)
+{
+    (void)ListTransactions(pwallet, wtx, nMinDepth, fLong, ret, filter_ismine, filter_label, -1);
```

Comment:
Why not just set a default value for limit above?

Reply:
Good catch, will do!